core/models/access.py

```diff
@@ -250,6 +250,12 @@ def is_active(self):
 
         return True
 
+    def display_name(self) -> str:
+        return f"{self.status}|{self.dataset}|{self.user or self.contact}"
```

Comment:
Should a space be added before and after the "|"? that is how it currently looks
<img width="348" alt="Screenshot 2023-11-21 at 12 10 56" src="https://github.com/elixir-luxembourg/daisy/assets/33813190/690bb19c-4348-447d-aabe-bb014214136a">

and that is how it would look with a space.
<img width="342" alt="Screenshot 2023-11-21 at 12 12 42" src="https://github.com/elixir-luxembourg/daisy/assets/33813190/790842f5-5c90-4efd-9184-44fec01f9a28">